examples/reuploading_classifier/qlassifier.py

```diff
@@ -2,7 +2,6 @@
 from qibo import gates
 import numpy as np
 from datasets import create_dataset, create_target, fig_template, world_map_template
-from qibo.config import matrices
```

Comment:
General remark, I think we should reconsider all examples and use K instead of manual numpy imports.